scripts/jira_tickets/merged_release_graph.js

```diff
@@ -95,7 +95,7 @@ function process_tickets(json) {
 
 
 // fetch ticket status from REST API
-var endpoint_ticket_list = 'https://www.ebi.ac.uk/panda/jira/rest/api/2/search/?jql=project=ENSCOMPARASW+AND+fixVersion="Ensembl+__RELEASE__"+AND+component+IN+("Relco+tasks","Production+tasks")+AND+labels+IS+NOT+EMPTY+ORDER+BY+created+ASC,id+ASC&maxResults=100';
+var endpoint_ticket_list = 'https://www.ebi.ac.uk/panda/jira/rest/api/2/search/?jql=project=ENSCOMPARASW+AND+(+fixVersion="Ensembl+__RELEASE__"+OR+fixVersion="Release+__RELEASE__"+)+AND+component+IN+("Relco+tasks","Production+tasks")+AND+labels+IS+NOT+EMPTY+ORDER+BY+created+ASC,id+ASC&maxResults=100';
```

Comment:
Alternatively, you could use the following syntax to make the URL more compact:
`(+fixVersion+IN+("Ensembl+__RELEASE__"+,+"Release+__RELEASE__")+)`
Although it is not that relevant as I don't expect the field `fixVersion` to be changed in the near future.